tests/test_io.py

```diff
@@ -130,9 +124,8 @@ def test_read_alias_derivative(self):
             dery = res["der(y)"]
 
```

Comment:
Should we also add `assert len(derx) > 0` to make sure the loop actually iterates over anything?

Reply:
Good catch, yes, we should absolutely test that; adding.